snippets/csharp/System/NullReferenceException/Overview/nullreturn2.cs

```diff
@@ -1,39 +1,43 @@
-﻿// <Snippet4>
+﻿
 using System;
 
-public class Example
+namespace NullReturn
 {
-   public static void Main()
-   {
-      Person[] persons = Person.AddRange( new String[] { "Abigail", "Abra",
+    public class NullReturnExample
+    {
+        // <Snippet4>
+        public static void Main()
+        {
+            Person[] persons = Person.AddRange([ "Abigail", "Abra",
                                           "Abraham", "Adrian", "Ariella",
-                                          "Arnold", "Aston", "Astor" } );
-      String nameToFind = "Robert";
-      Person found = Array.Find(persons, p => p.FirstName == nameToFind);
-      Console.WriteLine(found.FirstName);
-   }
-}
+                                          "Arnold", "Aston", "Astor" ]);
+            string nameToFind = "Robert";
+            Person found = Array.Find(persons, p => p.FirstName == nameToFind);
+            Console.WriteLine(found.FirstName);
+        }
+    }
 
-public class Person
-{
-   public static Person[] AddRange(String[] firstNames)
-   {
-      Person[] p = new Person[firstNames.Length];
-      for (int ctr = 0; ctr < firstNames.Length; ctr++)
-         p[ctr] = new Person(firstNames[ctr]);
+    public class Person
+    {
+        public static Person[] AddRange(string[] firstNames)
```

Comment:
This could be a params array (style only):

```suggestion
        public static Person[] AddRange(params string[] firstNames)
```